online-inference/fastertransformer/gptj-ft-inference-service.yml

```diff
@@ -0,0 +1,43 @@
+apiVersion: serving.kubeflow.org/v1beta1
+kind: InferenceService
+metadata:
+  labels:
+    qos.coreweave.cloud/latency: low
+  name: fastertransformer-triton
+spec:
+  predictor:
+    maxReplicas: 10
+    minReplicas: 1
+    containerConcurrency: 1
+    containers:
+    - name: gptj-6b-1-gpu-rtx-a5000
+      image: rtalaricw/gptj_ft:22.05
+      command: ["/opt/tritonserver/bin/tritonserver"]
+      args: ["--model-repository=/mnt/pvc/triton-model-store"]
+      # command: 
```

Comment:
As with the sleep pod do we want to leave this in the released example? Won't hurt anything, just only really needed for dev.